projects/multiple-network-demo/package.json

```diff
@@ -17,32 +17,29 @@
     "build": "yarn clean && yarn typecheck && webpack",
     "dev": "yarn clean && concurrently \"yarn typecheck:watch\" \"webpack serve --config webpack.config.js\"",
     "test": "exit 0; #No tests",
-    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "yarn eslint . --ext .ts,.tsx"
   },
   "devDependencies": {
-    "@types/jest": "^26.0.23",
-    "@typescript-eslint/eslint-plugin": "^5.5.0",
-    "@typescript-eslint/parser": "^5.5.0",
-    "concurrently": "^6.0.1",
-    "eslint": "^8.5.0",
-    "jest": "^27.0.5",
-    "ts-jest": "^27.0.3",
-    "typescript": "^4.5.2"
+    "@typescript-eslint/eslint-plugin": "^5.9.1",
+    "@typescript-eslint/parser": "^5.9.1",
+    "concurrently": "^7.0.0",
+    "eslint": "^8.6.0",
+    "typescript": "^4.5.4"
   },
   "dependencies": {
-    "@babel/core": "^7.15.8",
-    "@babel/preset-env": "^7.15.8",
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.8",
```

Comment:
since you are working on this, could you please move all the dev-dependencies to the `devDependencies` entry? :pray: (babel, weback, loaders, etc)

Reply:
Absolutely!!